server/database/get-user.js

```diff
@@ -1,12 +1,15 @@
 const dbClient = require('./db_connection.js');
+const uuidV1 = require('uuid/v1');
 
 const findUser = `
 SELECT * from users
 WHERE username = $1;`;
 
+const newId = uuidV1();
```

Comment:
Urm, what is this?

Each time you start the server you create this only once. Does it represent the server (what is going on)?